.github/workflows/test.yml

```diff
@@ -23,10 +23,10 @@ jobs:
       - run: pip install bandit black codespell flake8 mypy pyupgrade safety pylint
       - run: bandit -r . || true
       - run: black --check .
-      - run: codespell --ignore-words-list="mape" --quiet-level=2 --skip=".git" # --skip=""
-      - run: flake8 . --count --ignore=E203,E302,F401,W503 --max-line-length=140 --show-source --statistics
+      - run: codespell --ignore-words-list="mape" --quiet-level=2 --skip=".git"
+      - run: flake8 . --count --ignore=E203,E302,F401,W503 --max-line-length=122 --show-source --statistics
```

Comment:
out of curiosity, why 122 specifically?

Reply:
trial and error...  Fewer chars require too many changes.  The _gradual_ approach.